### ISSUE TITLE
Subtitle auto translation

### ISSUE THREAD

**Issue by nikhilCad** [TeamNewPipe/NewPipe#5338]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

If you open the Youtube website, there is an option to auto translate captions to your preffered language.
However,  as far as I can tell, there is no such option available on android or on m.youtube.com 
If it can be implemented, it will be great.

#### Is your feature request related to a problem? Please describe it

Some videos are in foreign language. I wanted to translate the subtitles to English for them.

#### Additional context

-

#### How will you/everyone benefit from this feature?

Language support always helps!

**Comment by Stypox** [comment]
I think implementing something like this in NewPipe would be too broad

**Comment by ephestione** [comment]
Hello, I think allowing for translation of auto generated captions would make a lot of useful content (in German, French, Russian, you name it) available to everyone.
Still, since this is not present in mobile YouTube I have no idea how work intensive the function could be for the developers.

**Comment by Pheggas** [comment]
Any update on this? I don't understand why @mhmdanas maked this feature request as completed when it's not done.

**Comment by mhmdanas** [comment]
Ignore the misleading "as completed" thing in GitHub's UI (which wasn't even a feature that existed when I closed this issue), I closed this because I (and @Stypox) believed it's out of scope for NewPipe.

**Comment by Pheggas** [comment]
How could it be out of scope when it's normal feature in the YouTube UI? I thought you want to move the NewPipe forward consistently.

**Comment by Stypox** [comment]
Yeah now that YouTube provides auto-translated subtitles themselves, this can be done, see https://github.com/TeamNewPipe/NewPipeExtractor/pull/997

**Comment by Pheggas** [comment]
Thank you for reopening. Looking forward for this feature in NewPipe!

**Comment by Baltazar500** [comment]
Will there be auto-translation into the selected language in the next release ? 

p.s. When will there be a new release ?

**Comment by Julmik647** [comment]
> However, as far as I can tell, there is no such option available on android or on m.youtube.com If it can be implemented, it will be great.

Update: this option is now avaiable on [m.youtube.com](https://m.youtube.com) and it's alo in Android app, but supports less langues than a website.
